crates/libcgroups/src/common.rs

```diff
@@ -216,6 +216,21 @@ fn create_v2_cgroup_manager(_cgroup_path: PathBuf) -> Result<Box<dyn CgroupManag
     bail!("cgroup v2 feature is required, but was not enabled during compile time");
 }
 
+pub fn rootless_required() -> bool {
```

Comment:
Can I ask you to write a unit test? The path(`/proc/self/uid_map`) should be tested by creating a function that returns the path and separation it from the test version, like that.
https://github.com/containers/youki/blob/95d85c510f94b701c3266728edcd68d4471e80e0/crates/youki/src/main.rs#L180-L188

Reply:
Sure thing, I'll add a unit test in the next update.

Reply:
FYI: you missed the doc comment when moving the function here. :wink: